app/src/main/java/net/squanchy/imageloader/ImageLoaderComponent.java

```diff
@@ -0,0 +1,12 @@
+package net.squanchy.imageloader;
+
+import net.squanchy.injection.ActivityLifecycle;
+
+import dagger.Component;
+
+@ActivityLifecycle
```

Comment:
It takes a Context, seems the logical thing to do to avoid leaks (and no, I don't think it should be an Application Context)